bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/Ostgotatrafiken.java

```diff
@@ -90,18 +86,19 @@ protected LoginPackage preLogin() throws BankException, IOException {
                 R.raw.cert_ostgotatrafiken_login, R.raw.cert_ostgotatrafiken_overview));
 
         List<NameValuePair> postData = new ArrayList<NameValuePair>();
-        postData.add(new BasicNameValuePair("Username", getUsername()));
-        postData.add(new BasicNameValuePair("Password", getPassword()));
-        postData.add(new BasicNameValuePair("Login", "Logga in"));
+        postData.add(new BasicNameValuePair("", "{\"authSource\":10,\"keepMeLimitedLoggedIn\":true," + 
```

Comment:
Since you only use json in this small part I can approve that you do only handle it as a string. However, I suggest that you use `String.format` instead to keep is a little cleaner